cfgov/jinja2/v1/_includes/macros/video-player.html

```diff
@@ -27,27 +27,30 @@
 
 {% macro render(options={}) -%}
     {% import 'macros/share.html' as share with context %}
-    {% set video_player = options.video.player | default('youtube') %}
-    {% set video_id     = options.video.id     | default('') %}
-    {% set video_url    = options.video.url    | default('/') %}
+    {% set video_url    = options.video.url | default('/') %}
+    {% set video_player = options.video.player or
+                          ( video_url.find( 'youtube.com') > -1 and 'youtube' ) or
+                          ( video_url.find( 'ustream.tv') and 'ustream' ) %}
+    {#  TODO: Replace with django method for getting video id if youtube #}
+    {% set video_id     = options.video.id or ( video_url.split('?')[0].split('/') | last )
```

Comment:
Alternatively, the `video_player` type check could be combined, but it's more code so I don't know which is better. Also, looks like this is changing in the future anyway. 

```
{% if video_player == 'youtube' %}
    {% set video_id = options.video.id or ( video_url.split('?')[0].split('/') | last ) %}
    {% set image_url = options.image.url or ( 'https://img.youtube.com/vi/' + video_id + '/maxresdefault.jpg' ) %}
{% elif %}
    {% set video_id = '' %}
    {% set image_url = '/static/img/cfpb_video_cover_card_1380x776.png' %}
{% endif %}
```